site/en/docs/privacy-sandbox/fledge/index.md

```diff
@@ -154,9 +147,55 @@ API callers can't access group membership when users browse in Incognito mode, a
 removed when users clear their site data.
 
 
-{: #why}
+## How can I opt out of FLEDGE? {: #opt-out}
+
+You can block access to the FLEDGE API either as a site owner, or as an individual user.
+
+### How can sites control access? {: #opt-out-site}
+
+FLEDGE will eventually require sites to set permissions policy to allow FLEDGE functionality to be
+available. This will help ensure that arbitrary third parties can't use the API without a site's
```

Comment:
```suggestion
FLEDGE will eventually require sites to set a [Permissions Policy](/docs/privacy-sandbox/permissions-policy/)
to allow FLEDGE functionality to be available. This will help ensure that arbitrary third parties can't use the API without a site's
```